v22.1/cockroach-sql-binary.md

```diff
@@ -0,0 +1,828 @@
+---
+title: cockroach-sql
+summary: cockroach-sql is a client for executing SQL statements from an interactive shell or directly from the command line.
+toc: true
+docs_area: reference.cli
+---
+
+{% assign release = site.data.releases | where_exp: "release", "release.major_version == page.version.version" | last %}
+
+The `cockroach-sql` command is a client for executing SQL statements from an interactive shell or directly from the command line. To use this client, run `cockroach-sql` as described below.
+
+{{site.data.alerts.callout_info}}
+`cockroach-sql` is functionally equivalent to the [`cockroach sql` command](cockroach-sql.html).
+{{site.data.alerts.end}}
+
+To exit the interactive shell, use `\q`, `quit`, `exit`, or `ctrl-d`.
+
+## Install `cockroach-sql`
+
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="linux"><strong>Linux</strong></button>
+    <button class="filter-button page-level" data-scope="mac"><strong>Mac</strong></button>
+    <button class="filter-button page-level" data-scope="windows"><strong>Windows</strong></button>
+</div>
+
+Download the binary and copy it into your `PATH`.
+
+<div class="filter-content" markdown="1" data-scope="linux">
```

Comment:
```suggestion
<div class="filter-content" markdown="1" data-scope="linux">

```

Reply:
I figured out the `copy-clipboard.html` problem. You need to add a line break after each `<div>` call. I think it's a JS thing.

Reply:
Nice. Done.